app/controllers/answers_controller.rb

```diff
@@ -21,6 +21,16 @@ def best
     flash[:notice] = 'Your choose best answer.' if @answer.choose_answer(@question)
   end
 
+  def like
```

Comment:
It would be great to extract these actions into a concern (let's call it `Voted`) and then just write `include Voted` in any controller which works with _votable_ entities.